### PR TITLE
fix: Sitemap not found during build

### LINKED ISSUE #207: Sitemap is missing
For some reason, the bjerk.io/sitemap.xml is not showing anymore. 🤷‍♂️

This is also causing our Lighthouse-checking to fail. We probably want to fix that 👍

### PR DESCRIPTION
**Pull request by ws333**
Updated push.yml

fixes #207

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -55,13 +55,13 @@ jobs:
 
       - uses: atymic/sitemap-ping-action@master
         with:
-          sitemap-url: https://bjerk.io/sitemap.xml
+          sitemap-url: https://bjerk.io/sitemap/sitemap-index.xml
```
Comment (cobraz) [inline .github/workflows/push.yml:58]: Why has this changed? Do we know?

Wondering if need to change some references elsewhere too. Might be some Google stuff that needs an update. 🤷‍♂️

**Review by cobraz — CHANGES_REQUESTED** [review]
.github/workflows/pull-request.yml needs an update too seem like

**Review by cobraz — APPROVED** [review]
LGTM!

File: .github/workflows/push.yml (hunk 1 above)
Comment (ws333) [reply .github/workflows/push.yml:58]: Could it be a `gatsby-plugin-sitemap` update that cause this?

**Pull request merged**